### ISSUE TITLE
Allow to suppress stderr of document viewer

### ISSUE THREAD

**Issue by driftregion** [xflr6/graphviz#72]
Hello,

With Python 3.7.3 on Ubuntu, calling `graphviz.files.File.view()` results in many messages from Evince document viewer (Ubuntu's default PDF viewer) being printed to stderr, like this:

```sh
** (evince:18324): WARNING **: 03:04:26.048: Error setting file metadata: can’t open metadata tree

** (evince:18324): WARNING **: 03:04:26.048: Error setting file metadata: can’t open metadata tree

** (evince:18324): WARNING **: 03:04:26.048: Error setting file metadata: can’t open metadata tree
```

This may be an issue with the verbosity of Evince rather than with graphviz.

At least one other user has had a similar issue: (not clear whether they're also using Evince)
https://stackoverflow.com/questions/53054200/how-to-suppress-graphviz-warnings-in-python-3/56393102#56393102

This issue appears related in topic: https://github.com/xflr6/graphviz/issues/57

**Comment by xflr6** [comment]
Thanks. I agree that this seems more of an [issue with Evince](https://www.google.com/search?q="Error+setting+file+metadata"). Adding a `quiet` parameter to `backend.view()`  (mirroring `backend.render()` and `backend.pipe()`, and making them available on `Graph` and `Source` instances) seems principally fine to me. However, I am  not sure how to suppress `stdout/err` for `backend.view()` under Windows, which uses `os.startfile()` instead of `subprocess.Popen` under the hood.

Note that the bug title, the SO-question, and the other issue you link to are about the `stderr` of the rendering process, not about the one from opening the viewer. Exposing the `quiet` parameter of `backend.render` on `Source.render()`, `Graph.render()`, etc. should be straightforward but won't help with your Evince warnings.

Can you try if changing  [this](https://github.com/xflr6/graphviz/blob/98ef97569138626fc0c1e4c11c00cfcf0777cc7d/graphviz/backend.py#L256) as follows would work for you?
```python
with open(os.devnull, 'w') as f:
    subprocess.Popen(['xdg-open', filepath], stderr=f)
```

**Comment by nkirkby** [comment]
Confirmed, after making this change exactly as you describe
```python
with open(os.devnull, 'w') as f:
    subprocess.Popen(['xdg-open', filepath], stderr=f)
```
The Evince warnings are redirected and no longer appear.

Got it, I also see now that the SO question pertains to rendering, not to opening the viewer.  Thanks for clarifying this.

**Comment by xflr6** [comment]
Added `quiet` argument to `.render()`  and  `.pipe()` methods in bfe1a9e1b04a6ebe4021be5b53f4926719bd99b9.

**Comment by xflr6** [comment]
Added a draft for ignoring viewer process `stderr` in https://github.com/xflr6/graphviz/tree/quiet_view (not sure about this part).

**Comment by xflr6** [comment]
[graphviz 0.11](https://pypi.org/project/graphviz/0.11/) adds `quiet` to `.render()`, `.pipe()`, and `.view()` methods.
Title updated for `stderr` of viewer process.

**Comment by xflr6** [comment]
Merged `quiet_view=True` to master via fa17c33e0ae8d3ec6c6d9214009ecc75dab3e433

**Comment by xflr6** [comment]
Fixed in [0.12 release](https://pypi.org/project/graphviz/0.12/#files): (00e5eccaf5f986995b9bf81464a6b338541317b2).
